examples/Jaeger.Example.WebApi/Jaeger.Example.WebApi.csproj

```diff
@@ -5,13 +5,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTracing.Contrib.NetCore" Version="0.6.2" />
+    <PackageReference Include="OpenTracing.Contrib.NetCore" Version="0.7.1" />
 
     <!-- NOTE:
         In case you're building this application standalone, use below PackageReference 
         with an appropriate version instead of relative-path ProjectReference 
         to Jaeger project -->
-    <!-- PackageReference Include="Jaeger" Version="0.2.1" /-->
+    <!-- PackageReference Include="Jaeger" Version="1.0.2" /-->
```

Comment:
This line is commented out in favor of the `ProjectReference` further down (which is used in CI).